core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -51,19 +54,26 @@
 import jakarta.validation.ConstraintViolationException;
 import java.time.format.DateTimeParseException;
 import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 /**
  * Custom exception handler.
  */
-@AllArgsConstructor
 @RestControllerAdvice
 @Slf4j
 public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
     private ErrorAttributes errorAttributes;
     private final ObjectMapper objectMapper;
+    @Value("${valid.endpoints}")
+    private List<String> validEndpoints;
```

Comment:
The exception handler is responsible for handling exceptions only
Please add sone validation filters for this logc